src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -761,6 +761,7 @@ private PackageDependency CreateDependencyFromProject(dynamic project, Dictionar
 
                 projectFactory.InitializeProperties(builder);
 
```

Comment:
I think you need to rebase the feature branch and then rebase this branch off of the feature branch.

This commit shouldn't be part of this PR.